ui/src/layout/Header.tsx

```diff
@@ -55,7 +56,7 @@ class Header extends Component<IProps> {
             <AppBar position="absolute" style={style} className={classes.appBar}>
                 <Toolbar>
                     <div className={classes.title}>
-                        <a href="https://github.com/gotify/server" className={classes.link}>
+                        <a href="/" className={classes.link}>
```

Comment:
I think this does a wrong redirect when gotify is served under a subpath (See f.ex https://gotify.net//docs/nginx#at-a-subpath). This can be fixed by using `<Link>` from `react-router-dom`.

See f.ex the "All Messages"-Link
https://github.com/gotify/server/blob/6edfd8400e58cbf51dda76d9e06ccac0471f1c71/ui/src/layout/Navigation.tsx#L68-L72

Reply:
Yeah, I think it's better for a correct routing.